hail/python/hail/docs/_static/rtd_modifications.css

```diff
@@ -2,24 +2,28 @@
 /* border-color on hail.is: #ddd */
 /* background-color on hail.is: #fff */
 .section {
-  margin-top: -51px;
+  margin-top: -50px;
+  padding-top: 50px;
 }
 
-.section:before {
-  height: 51px;
-  content: "";
-  display: block;
+dl+dl {
+  margin-top: 50px;
```

Comment:
this is a quick fix to the problem that some of these focusable blocks have no documentation, and therefore the space between them is smaller than the height needed for the offset, making them unclickable.

Reply:
ex: https://hail.is/docs/0.2/types.html#hail.expr.types.HailType.subst

Reply:
If we document each function I don't think the problem will emerge.

Reply:
that one shouldn't be documented at all.

Reply:
Should we have functions in our documentation that have no explanation?

Reply:
If you meant that even the function links shouldn’t be present I agree. Will make an issue

Reply:
#7292